src/main/java/com/google/step/snippet/external/Client.java

```diff
@@ -0,0 +1,9 @@
+package com.google.step.snippet.external;
+
+import com.google.step.snippet.data.Card;
+
+public interface Client {
+  Card search(String url);
+
+  String getCSEId();
```

Comment:
camelCase generally ignores the capitalization of abbreviations e.g. `getCseId()`. For example, you've been using `HttpClient` instead of `HTTPClient`.